deck/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class Deck(models.Model):
+    heros = ['druid', 'hunter', 'mage',
```

Comment:
This could probably be imported from a central file of some kind, right? It's small here, but I think better we encourage to keep things DRY.